subsys/net/ip/rpl.c

```diff
@@ -2044,7 +2044,7 @@ static bool net_rpl_process_parent_event(struct net_if *iface,
 	return ret;
 }
 
-static bool net_rpl_repair_root(u8_t instance_id)
+bool net_rpl_repair_root(u8_t instance_id)
```

Comment:
Could you merge the 2 first patches in one then? I don't see why it has to be separated, both follows the same exact need.